src/main/resources/db/migration/V1_17__alter_jrd_tables.sql

```diff
@@ -27,3 +27,5 @@ DROP table contract_type;
 ALTER TABLE base_location_type DROP COLUMN bench;
 ALTER TABLE base_location_type DROP COLUMN national_court_code;
 
+ALTER TABLE judicial_office_authorisation ADD COLUMN service_code varchar(16);
```

Comment:
Flyway rule is always add any new change in new flyway version file, otherwise on higher env migration will fail.